src/renderer/api/api-manager.ts

```diff
@@ -23,6 +23,7 @@ export class ApiManager {
       return this.apis.get(pathOrCallback) || this.apis.get(KubeApi.parseApi(pathOrCallback).apiBase);
     }
 
+    pathOrCallback ??= (api: KubeApi) => true; // pick first if none given
```

Comment:
Btw I would not overuse this operator(s) `??=` since it's not adds readability to code.

Reply:
I encountered a runtime error where this variable was not set to anything. Namely I got "pathOrCallback is not a function" (it was undefined) so I added this to return a better result.

@ixrock I don't think this is an overuse. This is exactly what that operator was designed for, plus I think it is much clearer than using `||=` for instance.

Reply:
Maybe better to change `first` to `true` inside a comment since first meaning is confusing (it can relate to first part of these operators, left-hand side)